jira-automation/migrations/sync-all-issues-from-jira-to-github.mjs

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+
+import { Octokit } from "@octokit/rest";
+import {jiraToGitHub} from "../jira-to-github-processing.mjs"
+import { getIssue } from "../jira.mjs";
+import { updateGitHubIssue } from "../github.mjs";
+
+const token = process.env.GITHUB_TOKEN || process.env.GITHUB_REPO_TOKEN;
+
+
+// Create an Octokit instance with your GitHub personal access token
+const octokit = new Octokit({
+  auth: token
+});
+
+// Fetch all open issues from the repository
+async function getOpenIssues() {
+  const response = await octokit.paginate(octokit.issues.listForRepo, {
+    owner: "hmcts",
+    repo: "roadmap-platform-operations",
+    state: "open",
+  });
+
+  // return response;
+  return response;
+}
+
+async function run() {
```

Comment:
do we need to write tests for this new module? I can see we have written tests for `jira-to-github-processing.mjs`

Reply:
this is in theory a one-off migration, not too valuable imo and it's calling common code, very little new code in here.